common.lic

```diff
@@ -590,6 +590,12 @@ module DRC
 
   def play_song?(settings, song_list, worn = true, skip_clean = false, climbing = false)
     instrument = worn ? settings.worn_instrument : settings.instrument
+    if UserVars.instrument != instrument
+      message("Instrument change detected. Resetting stored song data")
```

Comment:
What do you think about mentioning the old & new instruments?

```suggestion
      message("New instrument (#{instrument) detected; old instrument: #{UserVars.instrument}. Resetting stored song data")
```

Reply:
Great suggestion. Thank you!